sonar-db/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -560,6 +567,7 @@ public void insert() {
 
     ComponentDto componentDto = new ComponentDto()
       .setUuid("GHIJ")
+      .setUuidPath("ABCD.EFGH.GHIJ.")
```

Comment:
Ok, forget it, that's the current UUID which is omitted in the path, I was still on an old paradigm